src/components/Slider/Slider.scss

```diff
@@ -0,0 +1,67 @@
+// Definition of basic slider variables
+$slider-rail-height: 8px;
+$slider-handle-size: 24px;
+$slider-handle-border-width: 6px;
+
+// Centering adjustments of the handle on the rail
+$slider-handle-adjust-left: -($slider-handle-size / 2);
+$slider-handle-adjust-top: -(($slider-handle-size / 2) - ($slider-rail-height / 2));
+
+// Definitions of variables for paddings, shadows, etc.
+$slider-padding: 5px 0;
+$slider-box-shadow: 0px 1px 3px 0px rgba(0, 0, 0, .2);
+$slider-handle-box-shadow: 0px 2px 4px 0px rgba(0, 0, 0, 0.5);
+
+:global .rc-slider-track {
+    position: absolute;
+    height: $slider-rail-height;
+    border-radius: $slider-rail-height / 2;
+    background-color: var(--color-brand);
+    transition: background-color var(--transition-duration) ease;
+    box-shadow: $slider-box-shadow
+}
+
+:global .rc-slider-handle {
+    position: absolute;
+    width: $slider-handle-size;
+    height: $slider-handle-size;
+    margin-left: $slider-handle-adjust-left;
+    margin-top: $slider-handle-adjust-top;
+    border: solid $slider-handle-border-width white;
+    border-radius: 50%;
+    transition: background-color var(--transition-duration) ease;
+    background-color: var(--color-brand);
+    box-shadow: $slider-handle-box-shadow;
+
+    &:focus {
+        outline: none;
+    }
+}
+
+:global .rc-slider {
+    position: relative;
+    height: $slider-rail-height;
+    padding: $slider-padding;
+    width: 100%;
+
+    &:hover .rc-slider-track {
+        background-color: var(--color-brand-40);
+    }
+
+    &:hover .rc-slider-handle {
+        background-color: var(--color-primary);
+    }
+
+    * {
+        box-sizing: border-box;
+    }
+}
+
+:global .rc-slider-rail {
+    position: absolute;
+    width: 100%;
+    background-color: white;
+    height: $slider-rail-height;
+    border-radius: $slider-rail-height / 2;
+    box-shadow: $slider-box-shadow
+}
```

Comment:
mind that sass variables will not be themeable. If some of these vars supposed to be a part of theme roller, then they should be exposed and referenced as css-variables.
Feels like size and shadows related vars are good candidates for it.

Reply:
@antipin I discussed that issue with David and since there're no any particular color options in the design document for other themes and selected colors are from different color palettes, he decided to leave it as it is for now until Carlo is back in the office so we can get it clarified.